NetSuite Bundle update blog post (#851)
codatio/codat-docs: blog/231026-netsuite-bundle-update.md

---
title: "New NetSuite bundle released"
date: "2023-10-26"
tags: ["NetSuite", "Update"]
authors: todorlizhev
---

We have released a new version of our NetSuite bundle that contains fixes to the display of attachments. 

<!--truncate-->

> Previously, users may have experienced an issue where attachments using a `month/day/year` format weren't displaying their `createdDate` and `modifiedDate`. The new v1.23.0 version of the bundle corrects this behavior so the attachments display the `createdDate` and `modifiedDate`.

Reviewer: ```suggestion
Previously, users may have experienced an issue when viewing attachments using the `month/day/year` format for `createdDate` and `modifiedDate`. Version v1.23.0 improves support by displaying an attachments `createdDate` and `modifiedDate`.
```
